### ISSUE TITLE
Training images dimesions and effect on output

### ISSUE THREAD

**Issue by hsein777** [fizyr/keras-retinanet#1568]
hello, I'd like to know if what I am doing is practically applicable and if I will get results. I am training a model to detect plant diseases from the shapes of the leaves. if the input images are only images of the leaves themselves and small sizes about 250x250, will I be able to get good results if the testing images are of a full plant with multiple leaves and bigger dimensions?  
I posted an example of training and testing images.

if its is not the right training set images for the testing images, should I bring images similar to the testing ones (large and with background) and label them myself?

![0eacfbb6-3d68-4a89-b8f8-6b2d328102af___RS_HL 7961](https://user-images.githubusercontent.com/65351288/150117477-62239cfd-0aad-4ff8-8531-d1e9901e1297.JPG)

![download](https://user-images.githubusercontent.com/65351288/150117780-20241277-b5f6-40f4-9d6a-4ce8ee2ae9fe.jpg)
